provider/kubernetes/kubernetes.go

```diff
@@ -165,25 +166,16 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 					}
 				}
 
-				PassHostHeader := p.getPassHostHeader()
-
-				passHostHeaderAnnotation, ok := i.Annotations[types.LabelFrontendPassHostHeader]
-				switch {
-				case !ok:
-					// No op.
-				case passHostHeaderAnnotation == "false":
-					PassHostHeader = false
-				case passHostHeaderAnnotation == "true":
-					PassHostHeader = true
-				default:
-					log.Warnf("Unknown value '%s' for %s, falling back to %s", passHostHeaderAnnotation, types.LabelFrontendPassHostHeader, PassHostHeader)
-				}
+				PassHostHeader := getRulePassHostHeader(i, p)
+				PassTLSCert := getRulePassTLSCert(i, p)
```

Comment:
The two variables probably don't need to be exported. Can we decapitalize them?

Reply:
Done.